pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "How the W3C Web of Things brings JSON Schema to the Internet of Things"
+date: 2022-08-17T06:00:00+00:00
+type: Case Study
+authors:
+  - name: Juan Cruz Viotti
+    photo: /img/avatars/jviotti.webp
+    link: https://twitter.com/jviottidc
+excerpt: "Using JSON Schema at the W3C Web of Things to counter the fragmentation of the Internet of Things"
```

Comment:
actually "countering the fragmentation" is something we want to avoid since it starts with a negative tone. I would propose (from our input document):

> Using JSON Schema at the W3C Web of Things to create an interoperability layer so that different IoT platforms, protocols and standards can operate together.

I hope it is not too late to fit into the small part that is under the title

Reply:
Makes sense! Done